guides/common/assembly_renaming-server-or-smart-proxy.adoc

```diff
@@ -1,4 +1,13 @@
-include::modules/con_renaming-server-or-smart-proxy.adoc[]
+[id="Renaming_Server_or_Smart_Proxy_{context}"]
+= Renaming {ProjectServer} or {SmartProxyServerTitle}
+
+To rename {ProjectServer} or {SmartProxyServer}, use the `{project-change-hostname}` script.
+
+[NOTE]
+====
+When changing the {ProjectServer} or {SmartProxyServer} DNS domain name, consider updating the hostname to match by using `{project-change-hostname}`.
```

Comment:
This reads a bit weird. Maybe "when changing the DNS record for ..."? 

Also, it "NOTE" the best admonition? I'd either drop it or use "CAUTION" because it _can_ lead to issues.

Reply:
I can't use CAUTION (not fully supported by Red Hat Customer Portal), but I'll either drop it or use IMPORTANT.